serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -143,34 +139,41 @@ For more information, see <<proc-install-knative-minikube, Installing Knative on
 ----
 eval $(minikube -p minikube docker-env --profile knative)
 ----
+--
 
-When using Minikube, you can share your local registry with the Docker Daemon.
-
-NOTE: If your Serverless Workflow application container image was built before configuring Docker, then you might need to build the image again so that the image is available on the Minikube Docker.
+. Build your application to store it into the Minikube registry
++
+--
+If your Serverless Workflow application container image was built before configuring Docker to use in-cluster Docker daemon, then you might need to build the image again so that the image is available in the Minikube registry.
 
-Ensure that you use the following system property when building native container images that use Minikube Docker Daemon:
+[NOTE]
+====
+If you are building native container images, ensure that you use the following system property to use Minikube Docker Daemon:
 
 .System property to build container images
 [source,shell]
 ----
 -Dquarkus.native.remote-container-build=true
 ----
---
+====
 
-. Use the `-Dquarkus.container-image.registry=some_of_the_values_above` property to enable Knative fetch the container images from Minikube Docker Daemon.
-+
---
-You might be required to tag the container images using one of the following registry options:
+[IMPORTANT]
+====
+You might be required to tag the container images using one of the following registry:
 
 * `ko.local`
 * `dev.local`
 
-You might need to set the `imagePullPolicy` to `Never` or `IfNotPresent`, otherwise Minikube pulls the images from a remote registry. This behavior can be avoided by tagging the image using previously listed domains.
-
 For more information, see link:{knative_issue_url}[How to use locally built docker image].
+
+In that case, use the `-Dquarkus.container-image.registry=some_of_the_values_above` property to enable Knative fetch the container images from Minikube Docker Daemon.
+
+If you don't use those values, you might need to set the `imagePullPolicy` to `Never` or `IfNotPresent`, otherwise Minikube pulls the images from a remote registry. 
```

Comment:
```suggestion
If you do not use the values, you might need to set the `imagePullPolicy` to `Never` or `IfNotPresent`, otherwise, Minikube pulls the images from a remote registry. 
```